tree/dataframe/src/RDFNodes.cxx

```diff
@@ -89,10 +95,7 @@ std::string RCustomColumnBase::GetName() const
    return fName;
 }
 
-void RCustomColumnBase::InitNode()
-{
-   fLastCheckedEntry = std::vector<Long64_t>(fNSlots, -1);
-}
+void RCustomColumnBase::InitNode() {}
```

Comment:
this can potentially be problematic: we need to reset `fLastCheckedEntry` before the next event loop that this `RCustomColumnBase` is going to participate in, or we might risk to not update the custom column in rare cases (which would be a nightmare to debug :P )

Reply:
Solved (should have authored you in this commit, too :1st_place_medal:)